elections/uk/lib.py

```diff
@@ -24,38 +26,43 @@ def shorten_post_label(post_label):
     'party_ec_id',
 ]
 
-def get_extra_csv_values(person, election):
-    theyworkforyou_url = ''
+def get_extra_csv_values(person, election, post):
+    gss_code = ''
     parlparse_id = ''
-    try:
-        i = person.identifiers.get(scheme='uk.org.publicwhip')
-        parlparse_id = i.identifier
-        m = re.search(r'^uk.org.publicwhip/person/(\d+)$', parlparse_id)
-        if not m:
-            message = "Malformed parlparse ID found {0}"
-            raise Exception(message.format(parlparse_id))
-        theyworkforyou_url = 'http://www.theyworkforyou.com/mp/{0}'.format(
-            m.group(1)
-        )
-    except Identifier.DoesNotExist:
-        pass
-    candidacy = MembershipExtra.objects \
-        .select_related('base', 'base__post', 'base__post__area') \
-        .get(
-            election=election,
-            base__person=person,
-            base__role=election.candidate_membership_role,
-        )
-    post = candidacy.base.post
-    party = candidacy.base.on_behalf_of
-    try:
-        party_ec_id = party.identifiers.get(scheme='electoral-commission').identifier
-    except Identifier.DoesNotExist:
-        party_ec_id = ''
-    try:
-        gss_code = post.area.other_identifiers.get(scheme='gss')
-    except Identifier.DoesNotExist:
-        gss_code = None
+    theyworkforyou_url = ''
```

Comment:
Like the other one, I think that doesn't matter because of being in the `uk` app